Plugins/Public/event/Main.cpp

```diff
@@ -556,16 +583,16 @@ void LoadSettings()
 	ConPrint(L"EVENT DEBUG: Loaded %u event player data\n", iLoaded3);
 
 	LoadIDs();
-
+	HkLoadStringDLLs();
```

Comment:
Please move into DLL_PROCESS_ATTACH section of DllMain(), and add HkUnloadStringDLLs(); to the DETACH section